hub3/models/store_test.go

```diff
@@ -14,28 +14,28 @@
 
 package models
```

Comment:
Do we need to keep this?

Reply:
I prefer to remove this later when we remove hub3/model package.